crates/core/tedge/src/cli/mqtt/cli.rs

```diff
@@ -21,6 +21,9 @@ pub enum TEdgeMqttCli {
         /// QoS level (0, 1, 2)
         #[structopt(short, long, parse(try_from_str = parse_qos), default_value = "0")]
         qos: QoS,
+        /// Retain flag
+        #[structopt(short, long = "retain")]
```

Comment:
We are missing a default value here (should be false).

Reply:
Tried that, but got an error "message: default_value is meaningless for bool".